docs/source/Linux_Install_Guide.rst

```diff
@@ -237,8 +237,12 @@ dlrm/dlrm_dot_test-*                   a DLRM implementation using rocWMMA API
 dlrm/dlrm_dot_lds_test-*               a DLRM implementation using rocWMMA API with LDS shared memory
 gemm/mma_sync_test-*                   a simple GEMM operation [D = alpha * (A x B) + beta * C] using rocWMMA API 
 gemm/mma_sync_multi_test-*             a modified GEMM operation, each wave targets a sub-grid of output blocks using rocWMMA API
+gemm/mma_sync_multi_ad_hoc_test-*      an adhoc version of mma_sync_multi_test-*
 gemm/mma_sync_multi_lds_test-*         a modified GEMM operation, each wave targets a sub-grid of output blocks using LDS memory and rocWMMA API
```

Comment:
Some additional details:
gemm/mma_sync_multi_lds_test-*         a modified GEMM operation, each wave targets a sub-grid of output blocks using LDS memory and rocWMMA API and wave-level collaboration

gemm/mma_sync_coop_wg_test-*           a modified GEMM operation, each wave targets a sub-grid of output blocks using LDS memory and rocWMMA API and workgroup-level collaboration